packages/core/src/Serwist.ts

```diff
@@ -367,6 +381,48 @@ export class Serwist {
     });
   }
 
+  /**
+   * Registers static route rules using the experimental InstallEvent.addRoutes() API.
+   * These rules allow bypassing the service worker for specific requests to improve performance.
+   * 
+   * @param event - The install event from the service worker lifecycle
+   * @throws {Error} When the route rules are invalid
+   */
+  async registerRequestRoutes(event: InstallEvent): Promise<void> {
+    if (!this._requestRouteRules) {
+      return;
+    }
+
+    // Check if both the API and route rules are available
+    if (!event?.addRoutes) {
+      if (process.env.NODE_ENV !== "production") {
+        logger.warn("Static Routing API (InstallEvent.addRoutes) is not supported in this browser");
```

Comment:
I wonder how we can reference both "request rules" and "Static Routing API" in this log. Maybe "Request rules ignored as the Static Routing API is not supported in this browser"? Besides, it'd be nice to add a "." at the end to be consistent with the rest of the logs.